src/OddJobs/Types.hs

```diff
@@ -158,9 +172,10 @@ instance FromJSON Status where
       Left e -> fail e
       Right r -> pure r
 
-
 newtype JobRunnerName = JobRunnerName { unJobRunnerName :: Text } deriving (Eq, Show, FromField, ToField, Generic, ToJSON, FromJSON)
 
+newtype ResourceId = ResourceId { unResourceId :: Text } deriving (Eq, Show, FromField, ToField, Generic, ToJSON, FromJSON)
```

Comment:
TODO: Add a comment on the type explaining its purpose as an opaque identifier.

Reply:
Style nitpick - I've started preferring `rawX` instead of `unX`, so `rawResourceId`